recipes/xo/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "0.1.11" %}
+
+package:
+  name: xo
+  version: {{ version }}
+
+source:
+  fn: xo.tar.gz
+  url: https://github.com/scopatz/xo/archive/8f486a2709d376d00f98c09a86179bce2a99099a.tar.gz
```

Comment:
Sorry, tapped ctrl+enter by accident. I edited my last message, hope you got all of that.